python/tvm/relay/frontend/tflite.py

```diff
@@ -62,70 +62,72 @@ def __init__(self, model, subgraph, exp_tab):
         # Add more operators
         self.convert_map = {
             'ABS': self.convert_abs,
+            'ADD': self.convert_add,
+            'AVERAGE_POOL_2D': self.convert_average_pool2d,
+            'BATCH_TO_SPACE_ND': self.convert_batch_to_space_nd,
+            'CAST': self.convert_cast,
+            'CEIL': self.convert_ceil,
+            'CONCATENATION': self.convert_concatenation,
+            'CONV_2D': self.convert_conv2d,
+            'COS': self.convert_cos,
+            'DEPTHWISE_CONV_2D': self.convert_depthwise_conv2d,
+            'DETECTION_POSTPROCESS': self.convert_detection_postprocess,
+            'DIV': self.convert_div,
+            'ELU': self.convert_elu,
+            'EQUAL': self.convert_equal,
```

Comment:
Do you change it as alphabetical order?

Reply:
@FrozenGene Yes. its better to be in alphabetical order like TF. And it reduces the conflicts while merging PR.